modules/docs/lib/InformationHighlight/index.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {createContainer} from '@workday/canvas-kit-react/common';
+import {createModifiers, createStyles, cssVar} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+import {Base, BaseProps} from './Base';
+import {Heading} from './Heading';
+import {Body} from './Body';
+import {Icon} from './Icon';
+import {Link} from './Link';
+import {useInformationHighlightModel} from './modelHook';
+
+type Variant = 'emphasis' | 'caution' | 'attention';
+
+interface InformationHighlightProps extends BaseProps {
+  variant?: Variant;
+}
+
+const containerStyles = createStyles({
+  backgroundColor: base.soap100,
+  display: 'grid',
+  gridTemplateColumns: 'min-content',
+  columnGap: cssVar(system.space.x4, '1rem'),
```

Comment:
if you didn't add a fallback, cssVar isn't necessary here right?

Reply:
Correct. But I don't need the fallback.